pkg/antctl/antctl.go

```diff
@@ -509,6 +509,80 @@ var CommandList = &commandList{
 			},
 			transformedResponse: reflect.TypeOf(recordmetrics.Response{}),
 		},
+		{
+			use:   "update",
+			short: "Update parameters used in flow aggregator",
+			long:  "Update parameters used in flow aggregator. It includes logTicker.",
+			example: `  Update logTicker
+  $ antctl updateparam -l 2m
+  Update podLabels
+  $ antctl updateparam -p true
+  Update externalFlowCollectorAddr
+  $ antctl updateparam -e 
+  Update activeFlowRecordTimeout
+  $ antctl activeflowrecordtimeout -a 120s
+  Update inactiveFlowRecordTimeout
+  $ antctl inactiveflowrecordtimeout -i 240s`,
```

Comment:
all these examples seem wrong. Shouldn't it be `antctl update` instead of `antctl updateparam`?

Also `antctl update -p true` is not very readable. I don't think it makes sense to use a _flag_ for this. The command should probably be something like `antctl set recordContents.podLabels true`

Reply:
Thanks for reviewing. I forget to change the example section. 

> IMO, there is a _major_ issue with this PR:
For this part, I think you're right, it should be persistent. May I know how did you solve the similar issue in the past?

Reply:
We didn't solve this in the past. My proposal at the time was to have antctl mutate the ConfigMap and trigger a rolling update of the Antrea Deployment, but obviously it means that all the Antrea Agents had to be restarted. For the Flow Aggregator maybe it's not as much of an issue because it's a single Pod, but all the Agents will lose their connection to the FlowAggregator. Another solution is to have antctl mutate the ConfigMap and watch the ConfigMap from the K8s API in the FlowAggregator Pod. This is also not ideal because that's now how ConfigMaps are used typically.

Reply:
Does that make sense to let antctl mutate the ConfigMap and update the FA internal parameters at the same time? It seems in this way we can avoid restarting FA every time we update a parameter through antctl, and also have the update in the ConfigMap to make sure these changes does not loss after a restart.

Reply:
@yanjunz97 I have changed my current structure to the method you just mentioned. I'll update it to this PR later. I think it is more convenient if we can do both at the same time.

Reply:
@antoninbas @yanjunz97 
Thanks for the reviewing and suggestions.
Currently, I'll check if I can update the parameters by mutating the ConfigMap. Then I'll update the params to flow-aggregator at runtime. In this case, we don't need to trigger a rolling update of the flow-aggregator deployment. If the the fa pod restart, we still have our modification in the ConfigMap.
I also separate the update for clickhouse and flow-collector (see the top comment). Hope it is clear now.

Reply:
I think it makes more sense to update the ConfigMap, but let the FA Pod watch the ConfigMap and react to changes. This way we don't need to support extra APIs to modify the parameters in FA. It also makes more sense to only modify parameters in one place (also for consistency), and this way we can mutate the configuration at runtime from outside of the FA Pod (since the ConfigMap can be modified from anywhere, with the necessary permissions of course).

Reply:
@antoninbas @yanjunz97 
I've updated the PR. I have a watcher to detect the ConfigMap changes from FA Pod. Antctl now will only mutate the ConfigMap. The only concern is, since some variables are commented in the ConfigMap, I have to give it a default value before I first update it in the ConfigMap (Like what it does in the options.go). I'm not sure if it is a good method or these is a better way to get the data for the first time.